client/my-sites/themes/theme-options.js

```diff
@@ -66,10 +73,10 @@ const activate = {
 	action: activateAction,
 	hideForTheme: ( state, themeId, siteId ) => (
 		! getCurrentUser( state ) ||
-		( isJetpackSite( state, siteId ) && isJetpackSiteMultiSite( state, siteId ) ) ||
+		( config.isEnabled( 'jetpack/pijp' ) && isJetpackSiteMultiSite( state, siteId ) ) ||
```

Comment:
This doesn't seem right -- the check used to hide the option for JP multi-site installs; that doesn't have to do with the feature flag...